docs/source/getting_started/getting_started.md

```diff
@@ -2,19 +2,22 @@
 
 Here we take a high-level look at how the system works. The API is divided into two parts: the back office and the front office.
 
-The back office system can be used to ingest your content into the system. Every piece of content you ingest is managed as one document within the Xayn system.
+The back office system can be used to ingest your content into the system. Every piece of content you ingest is stored as one document with one or more snippets.
 
-During ingestion, the system creates a mathematical representation of the document which is used to match the document to the user's interests and searches.
+A snippet is a bit of text which the user might search for. In the simplest case you only have one snippet per document in which case the snippet is roughly equivalent to the document. If you have such a use case you can mentally replace snippet with document in most of the remaining documentation. Through as there are limits on
```

Comment:
> A snippet is a bit of text which the user might search for.
I think it's more clear to present snippets as pieces or the whole content of a document.

```suggestion
A document is comprised of one or more snippets. Each snippet represents a piece of the content of the document. In the simplest case, you only have one snippet per document in such a case you can mentally replace snippet with document in most of the remaining documentation. Through as there are limits on
```